_SUMMARY.md

```diff
@@ -19,7 +19,7 @@
         * [Publishing Your Plugin](tutorials/build_system/deployment.md)
     * [Using DevKit](basics/getting_started/using_dev_kit.md)
         * [Setting Up a Development Environment](basics/getting_started/setting_up_environment.md)
-        * [Check Out And Build Community Edition](basics/checkout_and_build_community.md)
+        * [Check Out And Build Community Edition](https://github.com/JetBrains/intellij-community/blob/master/README.md)
```

Comment:
I think we should remove this reference entirely, because it's not part of this documentation and not a required step for setting up a plugin development environment.